common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java

```diff
@@ -165,7 +165,7 @@ public BrokerInteractiveTokenCommandParameters brokerInteractiveParametersFromAc
                 .extraQueryStringParameters(extraQP)
                 .authority(authority)
                 .scopes(scopes)
-                .clientId(AuthenticationConstants.Broker.ACCOUNT_CLIENTID_KEY)
+                .clientId(intent.getStringExtra(AuthenticationConstants.Broker.ACCOUNT_CLIENTID_KEY))
```

Comment:
Is this related to an issue, or should there be more in the commit message stating why this alteration needs to be made?

Reply:
Yes, we accidentally passed a hardcoded key into the clientId field, instead of the actual value.